styleguide.config.js

```diff
@@ -1,6 +1,11 @@
+const path = require('path')
+
 module.exports = {
-  ignore: ['**/App.js'],
+  ignore: ['**/App.js', '**/*.test.js', '**/styles/GlobalStyle.js'],
   defaultExample: true,
```

Comment:
Consider setting this to false since you added dummy data to your ArtistList.md 🙈